### ISSUE TITLE
cloud-buster: potential broken tool due of the python3.8 

### ISSUE THREAD

**Issue by Hax0rG1rl** [BlackArch/blackarch#2631]
[X] I am submitting a bug report for existing functionality that does not work as intended

----------------------------------------

### Bug description
cloud-buster seems to be broken 

### Steps to reproduce
- use cloud-buster cli and assess any domain behind cloudflare

cli: **cloud-buster --dept full <target domain>**

**Actual result:** Describe here what happens after you run the steps above (i.e. the buggy behaviour)

```
There was an error getting results
Traceback (most recent call last):
  File "/usr/lib/python3.8/runpy.py", line 193, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "/usr/lib/python3.8/runpy.py", line 86, in _run_code
    exec(code, run_globals)
  File "bust/__main__.py", line 112, in <module>
    main(args)
  File "bust/__main__.py", line 27, in main
    scan(args)
  File "bust/__main__.py", line 55, in scan
    found = sub_scan(buster, args, technique)
  File "bust/__main__.py", line 65, in sub_scan
    target_found = scan_technique()
  File "bust/buster.py", line 78, in scan_dnsdumpster
    records = results['dns_records']
TypeError: list indices must be integers or slices, not str
```

**Expected result:** Describe here what should happen after you run the steps above (i.e. what would be the correct behaviour)

- A user would expect an error-free output. 

### Screenshots

![image](https://user-images.githubusercontent.com/3704868/73688591-3f592d80-4731-11ea-9c43-d35299b574c5.png)

### Info for developers

Latest cloud-buster package from the last blackarch distribution.

**Comment by noraj** [comment]
- [x] Build the PKGBUILD `ba-dev -b`
- [x] Basic testing: `ba-dev -t -e cloud-buster -p cloud-buster-194.b55e4a1-1-any.pkg.tar.zst` --> no crash
    ```
    usage: bust [-h] [--scan [OPTION [OPTION ...]]] [--sub [SUBDOMAIN [SUBDOMAIN ...]]] [--dept DEPT] DOMAIN
    bust: error: the following arguments are required: DOMAIN
    ```
- [x] Test with same options with random domain: `ba-dev -t -e 'cloud-buster --dept full github.com' -p cloud-buster-194.b55e4a1-1-any.pkg.tar.zst` --> no crash
    ```
    ============ The =====,---/V\
    ==== CloudBUSTER ====~|  (o.o)
    === by @SageHack =====uu--uu

    [download] https://www.cloudflare.com/ips-v4
    [download] https://www.cloudflare.com/ips-v6

    [target] github.com
    * ip: 140.82.118.3 (CF no)
    * status: 301 Moved Permanently
    [error] not behind Cloudflare
    ```
- [x] Test with same options and a domain behind cloudflare: `ba-dev -t -e 'cloud-buster --dept full thepiratebay.org' -p cloud-buster-194.b55e4a1-1-any.pkg.tar.zst` --> no crash
    ```
    ============ The =====,---/V\
    ==== CloudBUSTER ====~|  (o.o)
    === by @SageHack =====uu--uu

    [download] https://www.cloudflare.com/ips-v4
    [download] https://www.cloudflare.com/ips-v6

    [target] thepiratebay.org
    * ip: 162.159.137.79 (CF yes RAY-55f7a7591d5cb7a5-CDG)
    * http: cloudflare - 301 Moved Permanently
    list index out of range
    [dnsdumpster] thepiratebay.org
    * ip: 162.159.137.79 (CF yes RAY-55f7a7591d5cb7a5-CDG)
    * http: cloudflare - 301 Moved Permanently
    [dnsdumpster] 162.159.137.79
    * ip: 162.159.137.79 (CF yes RAY-55f7aa9238becd8f-CDG)
    * http: cloudflare - 403 Forbidden
    [dnsdumpster] 104.27.216.28
    * ip: 104.27.216.28 (CF yes RAY-55f7aa926bb2b7bd-CDG)
    * http: cloudflare - 403 Forbidden
    [dnsdumpster] www.thepiratebay.org
    * ip: 162.159.138.79 (CF yes RAY-55f7aa92bdf2ee6d-CDG)
    * http: cloudflare - 301 Moved Permanently
    [dnsdumpster] m.thepiratebay.org
    ...
    ```

No bugs from **cloud-buster*.
After upgrading to python3.8 most of the python modules need to be rebuild if
they were built in a previous version, eg. 3.7.

> It's because Python installs its modules in /usr/lib/python$MAJOR.$MINOR, so each Python update means you have to rebuild all module packages so they are in the right place (and have compatible bytecode).

Basically doing this should work:

```
pacman -Rns cloud-buster
pacman -S cloud-buster
```

Sorry no the blackarch fault, it's how python works.
